applications/ShapeOptimizationApplication/python_scripts/value_logger_trust_region.py

```diff
@@ -75,25 +75,25 @@ def _WriteCurrentValuesToFile( self ):
         with open(self.complete_log_file_name, 'a') as csvfile:
             historyWriter = csv.writer(csvfile, delimiter=',',quotechar='|',quoting=csv.QUOTE_MINIMAL)
             row = []
-            row.append(str("{:>4d}".format(self.current_iteration)))
+            row.append("{:>4d}".format(self.current_iteration))
 
             objective_id = self.specified_objectives[0]["identifier"].GetString()
-            row.append(str("{:>20f}".format(self.value_history[objective_id][self.current_iteration])))
-            row.append(str("{:>12f}".format(self.value_history["abs_change_obj"][self.current_iteration])))
-            row.append(str("{:>12f}".format(self.value_history["rel_change_obj"][self.current_iteration])))
+            row.append(" {:> .5E}".format(self.value_history[objective_id][self.current_iteration]))
+            row.append("{:>12f}".format(self.value_history["abs_change_obj"][self.current_iteration]))
+            row.append("{:>12f}".format(self.value_history["rel_change_obj"][self.current_iteration]))
 
             for itr in range(self.specified_constraints.size()):
                 constraint_id = self.specified_constraints[itr]["identifier"].GetString()
-                row.append(str("{:>20f}".format(self.value_history[constraint_id][self.current_iteration])))
-                row.append(str("{:>20f}".format(self.communicator.getReferenceValue(constraint_id))))
-                row.append(str("{:>12f}".format(self.value_history["len_bar_cons"][self.current_iteration][itr])))
-                row.append(str("{:>12f}".format(self.value_history["adj_len_bar_cons"][self.current_iteration][itr])))
-
-            row.append(str("{:>12d}".format(self.value_history["bi_itrs"][self.current_iteration])))
-            row.append(str("{:>12f}".format(self.value_history["bi_err"][self.current_iteration])))
-            row.append(str("{:>17f}".format(self.value_history["test_norm_dX_bar"][self.current_iteration])))
-            row.append(str("{:>12f}".format(self.value_history["norm_dX"][self.current_iteration])))
-            row.append(str("{:>12f}".format(self.value_history["step_length"][self.current_iteration])))
+                row.append(" {:> .5E}".format(self.value_history[constraint_id][self.current_iteration]))
```

Comment:
I hope one day we will have the table logger properly working and we could simplify this